lib/Cargo.toml

```diff
@@ -88,7 +90,7 @@ spin = "0.9.8"
 ### eth
 
 evm = { version = "0.39", default-features = false, features = ["with-serde", "tracing"] }
-evm-runtime = "0.39.0"
+evm-runtime = { version  = "0.39.0" }
```

Comment:
We can just specify the version if no other requirements are needed

Reply:
Resolved in [7340314](https://github.com/DeFiCh/ain/pull/2850/commits/7340314660205f1a7c8d0b85a863d404d7975243)